1-js/02-first-steps/15-function-expressions/article.md

```diff
@@ -1,84 +1,84 @@
-# Function expressions
+# Wyrażenia funkcyjne
 
-In JavaScript, a function is not a "magical language structure", but a special kind of value.
+W JavaScripcie, funkcje nie są "magicznymi konstrukcjami językowymi", lecz po prostu rodzajem wartości.
```

Comment:
Please replace `lecz po prostu rodzajem wartości` by `lecz szczególnym rodzajem wartości` - it shouldn't change the context of the original sentence :)